terraform/environments/corporate-staff-rostering/iam.tf

```diff
@@ -9,9 +9,17 @@ resource "aws_iam_user" "mgn_user" {
 }
 #tfsec:ignore:aws-iam-no-user-attached-policies
 resource "aws_iam_user_policy_attachment" "mgn_attach_policy" {
-  #tfsec:ignore:aws-iam-no-user-attached-policies
+  #tfsec:ignore:aws-iam-no-user-attached-policies "This is a short lived user, so allowing IAM policies attached directly to a user."
   #checkov:skip=CKV_AWS_40: "Skipping as tfsec check is also ignored"
   user       = aws_iam_user.mgn_user.name
   policy_arn = "arn:aws:iam::aws:policy/AWSApplicationMigrationAgentInstallationPolicy"
 }
 
+#tfsec:ignore:aws-iam-no-user-attached-policies
+resource "aws_iam_user_policy_attachment" "mgn_attach_policy" {
```

Comment:
The policy attachment will need a different name :)

Reply:
just noticed, when running it locally :) thanks!